tests/test_flows.sh

```diff
@@ -211,15 +211,31 @@ test_channel_selection() {
 }
 test_client_capability_query() { 
     status "test client capability"
+    sta_mac=00:00:00:11:00:22
 
     check_error=0
-    send_CAPI_1905 ${GATEWAY} $mac_agent1 0x8009 "tlv_type,0x90,tlv_length,\
-0x000C,tlv_value,{$mac_agent1_wlan0 0x000000110022}"
+    check send_CAPI_command ${GATEWAY} "DEV_SEND_1905,DestALid,$mac_agent1,MessageTypeValue,0x8009,tlv_type,0x90,tlv_length,\
+0x000C,tlv_value,{$mac_agent1_wlan0 0x000000110022}" $redirect
```

Comment:
Please use a *different* sta_mac here than the one for the success scenario. This helps to make sure we parse out the correct log message. Cfr. #822